imports/ui/components/Card.jsx

```diff
@@ -11,16 +11,19 @@ import AddIcon from "@material-ui/icons/PlaylistAdd";
 import CheckedIcon from "@material-ui/icons/PlaylistAddCheck";
 import { connect } from "react-redux";
 import { changeRating } from "../actions/AppActions.js";
+import Lightbox from "react-image-lightbox";
+import "react-image-lightbox/style.css";
 
 class CardComponent extends Component {
   constructor(props) {
     super(props);
     this.state = {
       data: this.props.post,
       isFav: false,
-      imageSrc: `/images/` + this.props.post.name + `.png`,
+      imageSrc: this.props.post.imageSrc,
       showDetails: false,
-      isLogginIn: false
+      isLogginIn: false,
```

Comment:
What is this for? Only place declared is here.

Reply:
Not sure... removed!